### ISSUE TITLE
Support blender 4.0

### ISSUE THREAD

**Issue by draco1544** [LogicReinc/LogicReinc.BlendFarm#103]
Is possible to add blender 4.0 to the software ?

**Comment by LogicReinc** [comment]
Hey @draco1544 ,
BlendFarm should technically be forward compatible, but I tested it out and apparantly Blender 4.0 wasn't found.
After investigating, I found that apparantly Blender changed FTP for Blender 4.0.
Blendfarm gets its versions from here: https://download.blender.org/release/
Going to 4.0 will redirect to https://ftp.nluug.nl/, hence it wasn't found.

I have made a fixed locally, and see if I can push this out today but I can't guarantee it.
